ci/ci.py

```diff
@@ -283,8 +286,15 @@ def release(channel = None):
     call(cmd)
 
 
-def build_sdk():
-    call('python scripts/build.py build_sdk')
+def build_sdk(channel = None):
```

Comment:
Always specify a channel. build.py will default to "stable" if no channel is provided.